schemas/package.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: package
+  title: Package
+  group: 2
+  short: These fields contain information about an installed software package.
+  description: >
+    These fields contain information about an installed software package. It contains general information about a
+    package, such as name, version or size. It also contains installation details, such as time, URL or
+    user who installed the package.
+  type: group
+  fields:
+
+    - name: artifact_id
+      level: core
+      type: keyword
+      description: ID uniquely identifying the package accross different hosts.
+      example: 9b52cc0d83f794f6fc6190f603fa2f62dd9aa2017165d19ff1d62742955a7a05
+
+    - name: name
+      level: core
+      type: keyword
+      description: Package name
+      example: go
+
+    - name: version
+      level: core
+      type: keyword
+      description: Package version
+      example: 1.12.9
+
+    - name: description
+      level: core
+      type: keyword
+      description: Description of the package.
+      example: Open source programming language to build simple/reliable/efficient software.
+
+    - name: size
+      format: string
+      level: core
+      type: long
+      description: Package size in bytes.
+      example: 62.231
+
+    - name: installed
+      level: core
+      type: date
+      description: Time when package was installed.
+
+    - name: remote_repository
+      level: core
+      type: keyword
+      description: Remote repository URL from where the package was installed.
+      example: https://golang.org
+
+    - name: build_version
+      level: extended
+      type: keyword
+      description: Build version of the installed package, e.g. commit hash.
+      example: 9b52cc0d83f794f6fc6190f603fa2f62dd9aa2017165d19ff1d62742955a7a05
+
+    - name: build_time
+      level: extended
+      type: date
+      description: Time when package was built.
+
+    - name: checksum
+      level: extended
+      type: keyword
+      description: Checksum of the installed package
+      example: 9b52cc0d83f794f6fc6190f603fa2f62dd9aa2017165d19ff1d62742955a7a05
+
+    - name: install_type
+      level: extended
+      type: keyword
+      description: Indicating when the package was installed, e.g. at runtime or at buildtime.
```

Comment:
You're right, the field would make sense to use for distinguishing the install scope. Should we also rename to `install_scope` (e.g. user-local, global)?

Reply:
I see you've renamed to `install_scope` already, but I'll keep the discussion here for a bit ;-)

So `install_scope` makes tons of sense to me. I could see the following values in there:

* global
* user-local
* vendor (do you agree with this one?)

And I'd rather have the definition list them all up front (e.g. "The expected values are..."), so the field content is consistent across sources. 

WDYT? And do you agree with my current list of expected values, or do you see more?

Reply:
As long as we are open for _more_ values in the future, I think defining a starting list would be fine. I am not sure about `vendor`, as depending on the vendor mechanism this could also be system-wide for a specific version or local to a project.

Reply:
@webmat I think we need to clear this out - should I add the suggested list?

Reply:
The definition currently gives "user-local" and "global" as examples. I think that's enough for now.

If there's uncertainty about vendoring, we can skip for now, and we'll hash it out when it comes up with a specific use case. WDYT?

Reply:
I'm +1 on skipping it for now, we have no real-world usage examples. We can always add it later.